tests/appsec/appsec/pygoat_tests/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3"
```

Comment:
Can't we use the global docker-compose? Like
https://github.com/DataDog/dd-trace-py/blob/main/docker-compose.yml

Reply:
Done!